tests/ops/test_delta.py

```
@@ -79,45 +65,36 @@ def test_chunk(
-    reason='Skipping test_chunk_varlen because SKIP_TEST_CHUNK_VARLEN is set'
+@pytest.mark.parametrize(
+    ("cu_seqlens"),
+    [
+        ([0, 15, 100, 300, 1203, 2000]),
+    ]
 )
 @pytest.mark.skipif(
     device_platform == 'intel',
     reason='Intel Triton Failure'
 )
 def test_chunk_varlen(
```

Pylint (3.3.7)

[refactor] Too many local variables (28/15)

(R0914)